lib/tracing-limit/src/lib.rs

```diff
@@ -221,7 +224,7 @@ where
         ctx: &Context<S>,
         metadata: &'static Metadata<'static>,
         message: String,
-        rate_limit: std::num::NonZeroU64,
+        rate_limit: u64,
```

Comment:
It's still not valid for `rate_limit` to be zero, yes? Because of line 124 we never create a limit with a non-zero limit, if I read this correctly.

Reply:
Yeah, that's correct, line 120 will just short circuit rate limiting. This has the same effect as allowing it through, without the additional state manipulation.

Reply:
I expanded the comment to indicate this: d064c3f